src/components/transaction/Details.vue

```diff
@@ -155,13 +155,13 @@
       </div>
     </section>
 
-    <section v-if="isMultiSignature(transaction.type, transaction.typeGroup)" class="page-section py-5 md:py-10 mb-5">
+    <section v-if="isMultiSignature(transaction.type, transaction.typeGroup)" class="TransactionDetails__MultiSignature page-section py-5 md:py-10 mb-5">
       <div class="px-5 sm:px-10">
         <div class="list-row-border-b">
           <div class="mr-4">{{ $t("TRANSACTION.MULTI_SIGNATURE.ADDRESS") }}</div>
           <div class="truncate">
             <LinkWallet
-              :address="addressFromMultiSignatureAsset(transaction.asset.multiSignature)"
+              :address="addressFromMultiSignatureAsset(multiSignatureAsset)"
```

Comment:
the legacy multisig wallets did not generate a new wallet. Instead they made the sending wallet a multisignature one 🤔 So this now shows a wallet address for legacy multisig that does not exist

Reply:
Oh, I did not know that. I'll adjust it asap.

Reply:
Should be good to go now, see 713eb1a and 5e6e89d.

Reply:
The "Generated address" line will now show only for non-legacy transactions.